### DISCUSSION TITLE
I try to reverify my Education account but it doesn`t work.

### DISCUSSION THREAD

**Discussion by HCBlackFox** [community/community#68558] [GitHub Education]
### Hi everyone,

I try to reverify my education account after 2 years, but it doesn't work. I use academic email and also provide valide student ID but it keeps saying:

Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.
Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

What should I do with it?

**Comment by harxh-v** [comment]
Dear @Topdev512 

To address this issue, follow these steps:

1. **Update Your GitHub Profile**: Log in to your GitHub account > Go to your profile by clicking on your profile picture in the upper-right corner and selecting "Your profile." > Click the "Edit profile" button.

2. **Ensure Your Full Name Matches Academic Affiliation Document**: In the "Name" section, ensure that your full name exactly matches the name on your academic affiliation document, including any middle names or initials.
   
3. **Add a Short Bio** : Scroll down to the "Bio" section and enter a short bio about yourself.

4. **Save Changes**:  After making these updates, scroll to the bottom of the page and click the "Update profile" button to save your changes.

5. **Log Out and Log Back In**: Log out of your GitHub account, Log back in to ensure that the changes you made to your profile have taken effect.

6. **Reapply for Verification**: Once you've updated your profile information and logged out and back in, you can try to reapply for verification using your academic email and student ID.

Hope this helps! 👍

**Reply by HCBlackFox** [reply]
I've done this many times, these tips don't work... And not only for me, as can be seen from the comments

**Reply by harxh-v** [reply]
Yes, you are correct partially. As I have gone through their other community posts, they were not using academics email as their primary email. It is because GitHub has updated their t&c and onboarding terms.

If you are from one of them, change your primary email and then try-out.

**Reply by HCBlackFox** [reply]
I tried and nothing changed( I dont understand why someone cant manually review my request.

**Reply by harxh-v** [reply]
For your kind information, they no longer manually verify each request. If you have edu-email, you are welcome, if you don't- you can't join. Ask your college for an email.
Thank you!

**Reply by harxh-v** [reply]
Go through the updated [GitHub Education Terms](https://education.github.com/schools/terms).

Currently, they are only approving students with edu-email. They have paused onboarding as per any other verification methods.
There's nothing you can do, your re-verification request will be automatically rejected.

Kindly mark the post as answered.

**Reply by HCBlackFox** [reply]
But I have only my edu email in my account and it doesnt work lol...

**Comment by samus-aran** [comment]
Hi @HCBlackFox 👋 

Looking at your profile I suspect it's a translation issue. Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
